shared/cfg/machines.cfg

```diff
@@ -16,6 +16,14 @@ variants:
             cd_format = ahci
         enable_pvpanic = yes
         ioport_pvpanic = 0x505
+        pci_controllers = root_port pcie_switch dmi2pci_bridge pci_bridge
+        pci_bus = pci.0
```

Comment:
Yep, that would be my recommendation as the mainstream is currently pci so we should use pci-bridge as base.